### PR TITLE
feat($location):  add offset and limit args

### PR DESCRIPTION
**Pull request by jonashleyo**
Added "offset" and "limit" args to help review imports of FJC data.

### CODE REVIEW

File: cl/people_db/management/commands/cl_import_judges.py [hunk 1]
```diff
@@ -88,7 +102,11 @@ def import_fjc_judges(self, infile=None):
         df['Professional Career'].replace(to_replace=r';\sno', value=r', no',
                                             inplace=True, regex=True)
         for i, row in df.iterrows():
```
Comment (mlissner) [inline cl/people_db/management/commands/cl_import_judges.py]: Don't you need to have an `enumerate` around this to establish the index variable?

File: cl/people_db/management/commands/cl_import_judges.py [hunk 2]
```diff
@@ -88,7 +102,11 @@ def import_fjc_judges(self, infile=None):
...
-            make_federal_judge(dict(row), testing=self.debug)
+            if i < self.options['offset']:
+                make_federal_judge(dict(row), testing=self.debug)
```
Comment (mlissner) [inline cl/people_db/management/commands/cl_import_judges.py]: This isn't quite right. You `continue` if it's less than the offset, and if it's more, you do the work. 

So if the offset is 5, you skip the first five records (i.e., you `continue` until `i` is 6, *then* you do the work. If this doesn't make sense, grep the code for the word "offset" and you should see a handful of additional examples.

File: cl/people_db/management/commands/cl_import_judges.py (hunk 1 above)
Comment (jonashleyo) [reply cl/people_db/management/commands/cl_import_judges.py]: My understanding of pandas 'iterrows' is that it returns an iterator containing the index of each row and the data in a row as a Series.  I don't think it's necessary here but I can re-write if you want.

File: cl/people_db/management/commands/cl_import_judges.py (hunk 2 above)
Comment (jonashleyo) [reply cl/people_db/management/commands/cl_import_judges.py]: Just corrected and resubmitted this.

File: cl/people_db/management/commands/cl_import_judges.py (hunk 1 above)
Comment (mlissner) [reply cl/people_db/management/commands/cl_import_judges.py]: Right you are. I missed that this wasn't part of the diff.

File: cl/people_db/management/commands/cl_import_judges.py [hunk 3]
```diff
@@ -88,7 +102,12 @@ def import_fjc_judges(self, infile=None):
         df['Professional Career'].replace(to_replace=r';\sno', value=r', no',
                                             inplace=True, regex=True)
         for i, row in df.iterrows():
-            make_federal_judge(dict(row), testing=self.debug)
+            if i < self.options['offset']:
```
Comment (mlissner) [inline cl/people_db/management/commands/cl_import_judges.py]: This should just be:

```
if i < self.options['offset']:
    continue
if i >= self.options['limit'] > 0:
    break
make_federal_judge(dict(row), testing=self.debug)
```

Comment (jonashleyo) [reply cl/people_db/management/commands/cl_import_judges.py]: Shoot.  Resubmitted.

**Comment by mlissner** [comment]
Thanks Jon. Let me know next steps I should do if this is in my court.

**Comment by jonashleyo** [comment]
If you could try importing a few records (or all of them) I'll go through and double-check them for errors (if any).

**Pull request merged**